query/stdlib/testing/testing.go

```diff
@@ -1 +1,107 @@
 package testing
+
+var FluxEndToEndSkipList = map[string]string{
```

Comment:
Here I merged the 2 skip lists and tried to keep the most meaningful reasons.

Reply:
Merged from where?

Reply:
From the one that I removed here: https://github.com/influxdata/idpe/pull/3434

Reply:
I think it might be better if we just expose the skip list from influxdb and then in the enterprise product, we can just check both the influxdb skip list and the idpe specific-skip list.

Reply:
I'm doing it now